setup.py

```diff
@@ -44,16 +44,16 @@
     install_requires=['django-model-utils>=4.0.0,<4.1.0'],
     extras_require={
         'qa': [
-            'black<=19.10b0',
+            'black<=20.8b1',
             'flake8<=3.9',
             'isort~=5.0',
-            'readme-renderer~=26.0',
+            'readme-renderer~=28.0',
             'coveralls',  # depends on coverage as well
         ],
         'rest': [
-            'djangorestframework>=3.11,<3.12',
+            'djangorestframework~=3.12',
             'django-filter>=2.2.0<2.4.0',
-            'drf-yasg>=1.17,<1.18',
+            'drf-yasg~=1.20',
```

Comment:
are you sure there's no backward incompatible changes affecting us?

Reply:
This only updates requirements for itself no breaking changes mentioned in the changelog, check here: https://drf-yasg.readthedocs.io/en/stable/changelog.html